proposals/apple-mdm/base-setup-clix.md

```diff
@@ -0,0 +1,192 @@
+
+```
+
+## Misc
+### Error States
+#### Missing required flag
+Matches current behavior of `fleetctl`: Simply shows the help page. (To simulate this, try `fleetctl setup --name="hello"`)
+
+#### Required file does not exist 
+Matches current behavior of `fleetctl`: "`Error: open does/not/exist: no such file or directory`".
+
+(To simulate this, try `fleetctl apply -f does/not/exist`)
+### Help pages - Setup
+
+```
+$ fleetctl setup
+NAME:
+   fleetctl setup - Set up a Fleet instance or MDM // UPDATED
+
+USAGE:
+   fleetctl setup [options]
+   fleetctl setup apns // NEW
+   fleetctl setup scep // NEW
+   
+SUBCOMMANDS:
+	apns Apple Push Notification Service to enable MDM for Apple devices // New
+	scep Simple Certificate Enrollment Protocol to enable MDM for Apple devices // NEW
+
+OPTIONS:
+	... // remains same
+```
+
+### Help pages - APNS
+
+```
+// ALL NEW
+$ fleetctl setup apns
+NAME:
+	fleetctl setup apns - Set up Apple Push Notification Service to enable MDM for Apple devices
+
+USAGE:
+	fleetctl setup apns command
+
+COMMANDS:
+	init     Generate the necessary files to set up APNS
+	gen-req  Generate the .req file for Apple from the .p7 file from Fleet
+	finalize Set the SCEP certs in the Fleet Server's environment variables
+```
+
+```
+// ALL NEW
+$ fleetctl setup apns init
+NAME:
+	fleetctl setup apns init - Generates the necessary files to set up APNS 
+
+USAGE:
+	fleetctl setup apns init
+```
+
+```
+// ALL NEW
+$ fleetctl setup apns gen-req
+NAME:
+	fleetctl setup apns gen-req - Generate a .req file from a .p7 file provided by FleetDM, Inc.
+
+USAGE:
+	fleetctl setup apns gen-req [options]
+
+OPTIONS:
+  --p7-file value  Path to the .p7 file provided by FleetDM, Inc. (required)
+  --pki-key value  Path to the fleet-apns-pki.key file generated by the `fleetctl setup apns init` command (required)
+  --pki-crt value  Path to the fleet-apns-pki.crt file generated by the `fleetctl setup apns init` command (required)
+  --context value  Name of fleetctl config context to use (default: "default") [$CONTEXT]
+```
+
+```
+// ALL NEW
+$ fleetctl setup apns finalize
+NAME:
+	fleetctl setup apns finalize - Set the APNS certs in the Fleet Server's environment variables
+
+USAGE:
+	fleetctl setup apns finalize [options]
+
+OPTIONS:
+	--apple-crt value Path to the .crt file provided by Apple (required)
+	--push-key value  Path to the fleet-apns-push.key file generated by the `fleetctl setup apns init` command (required)
+  --context value   Name of fleetctl config context to use (default: "default") [$CONTEXT]
+```
+
+### Help Pages - SCEP
+
+```
+// ALL NEW
+$ fleetctl setup scep
+NAME:
+	fleetctl setup scep - Set up Simple Certificate Enrollment Protocol to enable MDM for Apple Devices
+
+USAGE:
+	fleetctl setup scep command
+
+COMMANDS:
+	init     Generate the necessary files to set up SCEP
+	finalize Set the SCEP certs in the Fleet Server's environment variables
+```
+
+```
+// ALL NEW
+$ fleetctl setup scep finalize
```

Comment:
This one can be removed now. (SCEP has only one `fleetctl` step, then it's just setting the env vars)